test/optional-validation.test.js

```diff
@@ -226,7 +226,7 @@ describe('optional-validation', function() {
 
       // backwards compatible with validateUpsert
       it('should NOT throw on updateOrCreate(id) with invalid data', function(done) {
-        callUpdateOrCreateWithExistingUserId(null, expectChangeSuccess(INVALID_DATA, done));
+        callUpdateOrCreateWithExistingUserId(null, expectValidationError(done));
```

Comment:
I find this change suspicious, isn't it a sign that this pull request may break existing applications?